creator-node/src/components/healthCheck/healthCheckController.js

```diff
@@ -42,52 +38,7 @@ const FIND_SYNC_REQUESTS_JOB_MAX_LAST_SUCCESSFUL_RUN_DELAY_MS = config.get(
 const FIND_REPLICA_SET_UPDATES_JOB_MAX_LAST_SUCCESSFUL_RUN_DELAY_MS =
   config.get('findReplicaSetUpdatesJobLastSuccessfulRunDelayMs')
 
-// Helper Functions
-/**
- * Verifies that the request is made by the delegate Owner
- */
-const healthCheckVerifySignature = (req, res, next) => {
```

Comment:
this was a test middleware which we had added but no longer necessary with ensureValidSPMiddleware